# string (topic name)
image_topic: "/zed/left/image_raw_color"  # Asus Xtion & Orbbec Astra Pro: "/camera/rgb/image_raw" # Stereolabs ZED: "/zed/left/image_raw_color" OR monochrome: "/zed/left/image_mono"
# string (topic name)
output_topic: "/openpose_ros/human_list"
# string (topic name)
depth_topic: "/zed/depth/depth_registered" # Asus Xtion & Orbbec Astra Pro: "/camera/depth/image" # Stereolabs ZED: "/zed/depth/depth_registered"
# boolean
display_output: true
# boolean
print_keypoints: false
# boolean
save_original_video: false
# boolean
save_openpose_video: false
# string (file name)
original_video_file_name: ""
# string (file name)
openpose_video_file_name: ""
# integer (number of frames per second)
video_fps: 10
# integer (number of positions)
depth_queue_size: 1
# integer (number of positions)
human_list_queue_size: 1